MoEngagePluginBase/MoEngagePlugin.swift

```diff
@@ -53,32 +59,39 @@ import MoEngageInApps
         initializeMoEngageSecondaryInstance(sdkConfig: sdkConfig, launchOptions: launchOptions)
     }
     
+    private func initializeMoEngageSecondaryInstance(sdkConfig: MOSDKConfig, sdkState: MoEngageSDKState) {
+       #if DEBUG
+         MoEngage.sharedInstance().initializeTestInstance(with: sdkConfig, andSDKState: sdkState)
+      #else
+        MoEngage.sharedInstance().initializeLiveInstance(with: sdkConfig, andSDKState: sdkState)
+      #endif
+        
+     commonSetUp(identifier: sdkConfig.identifier)
```

Comment:
check for indentation here @RakshithaAcharya

Reply:
done